osf/models/preprint_provider.py

```diff
@@ -14,21 +14,19 @@
 
 class PreprintProvider(ObjectIDMixin, BaseModel):
```

Comment:
It's going to be sent to SHARE as part of a query. We won't be querying inside of it, but we might need to use the individual strings. Comma delimiting will be easy to parse, though.